server/routes.go

```diff
@@ -4,13 +4,16 @@ import (
 	"context"
 	"encoding/json"
 	"net/http"
+	"os"
 	"strings"
 	"sync"
 
 	"github.com/gorilla/mux"
 	"github.com/sirupsen/logrus"
 )
 
+var ()
+
```

Comment:
I don't think that's intended do be here

Reply:
No it is not... Will remove when at pc...